app/javascript/close_container.js

```diff
@@ -0,0 +1,11 @@
+document.addEventListener("turbo:load", () => {
+  const containers = document.querySelectorAll('[data-component="closable"]');
+  console.log(containers)
+  containers.forEach(container => setupContainer(container))
+})
+
+const setupContainer = (container) => {
+  document.getElementById('close-btn').addEventListener('click', () => {
```

Comment:
This is wrong!
It should be only one **id** per HTML document. This means your there's no way to have more than one component per HTML document, meaning line 2 just makes no sense